redis_json/src/c_api.rs

```diff
@@ -267,6 +272,54 @@ pub fn json_api_is_json<M: Manager>(m: M, key: *mut rawmod::RedisModuleKey) -> c
     m.is_json(key).map_or(0, |res| res as c_int)
 }
 
+pub fn json_api_get_key_value<'a, M: Manager>(_: M, val: *const c_void) -> *const c_void
+where
+    M::V: 'a,
+{
+    let json = unsafe { &*(val.cast::<M::V>()) };
+    match json.get_type() {
+        SelectValueType::Object => {
+            let vec = json.items().unwrap().collect::<Vec<(&'a str, &'a M::V)>>();
```

Comment:
Same comment as above

Reply:
Was missing a `Box::new` in `json_api_get_key_value`